### ISSUE TITLE
XB_LEFT_TRIGGER and XB_RIGHT_TRIGGER trigger hotkey combinations by themselves

### ISSUE THREAD

**Issue by masterotaku** [bo3b/3Dmigoto#100]
Using 3Dmigoto 1.3.11 x64. While I was doing the update for my Dragon Ball Xenoverse 2 fix, I tried to make a "hold" hotkey for low convergence consisting of this:

Key = XB_LEFT_TRIGGER XB_RIGHT_TRIGGER XB_X

Pressing any of the triggers made the hotkey work, instead of having to press the full combination. The same happened after removing one of the triggers for the hotkey. Pressing the X button alone didn't trigger it, as expected.

However, things like this (and all face and/or shoulder button combinations) worked as intended:

Key = XB_LEFT_SHOULDER XB_X

In the end I settled for a different hotkey for now.

**Comment by DarkStarSword** [comment]
Can you check what happens if you put the non trigger button first, like XB_X XB_LEFT_TRIGGER XB_RIGHT_TRIGGER.

The triggers have some special parsing code to allow for different thresholds, which I think is taking priority over and disabling the key combination parsing.

**Comment by masterotaku** [comment]
OK, I'll try that. But you'll have to wait 7 or 8 hours for the results (I'm not at home right now).

**Comment by masterotaku** [comment]
Yes! That's the correct solution. Thanks, DSS. Having the hotkey like this is what I intended from the beginning:

Key = XB_X XB_RIGHT_TRIGGER XB_LEFT_TRIGGER
Key = XB_Y XB_RIGHT_TRIGGER XB_LEFT_TRIGGER
Key = XB_B XB_RIGHT_TRIGGER XB_LEFT_TRIGGER

**Comment by Darthagnon** [comment]
This answered a question I had (I was wondering why I couldn't get trigger+button hotkeys to work). Can this be put in the readme?
